CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 
 ### Added
 
+- [#121](https://github.com/jamwaffles/ssd1306/pull/121) Brightness control
```

Comment:
```suggestion
- [#121](https://github.com/jamwaffles/ssd1306/pull/121) Added brightness control with the `set_brightness()` method.
```

`set_brightness` here assuming you've switched `change_brightness` to `set_brightness` in the code.